web/cypress/integration/route/create-edit-route-with-redirect-plugin.spec.js

```diff
@@ -61,8 +61,8 @@ context('Create Edit and Delete Route with redirect plugin', () => {
     cy.get(selector.customRedirectLabel).should('be.visible');
     cy.get(selector.customRedirectUrI).should('be.visible');
     cy.get(selector.customRedirectCode).should('be.visible');
-    cy.get(selector.webSocketSelector).should('not.exist');
-    cy.get(selector.enable_websocket_button).should('not.exist');
+    cy.get(selector.webSocketSelector).should('exist');
+    cy.get(selector.enable_websocket_button).should('exist');
```

Comment:
would you please add an assertion to test enable websocket succseefully and check the info in configure view? @LiteSun